src/NUnitEngine/nunit.engine/nunit.engine.csproj

```diff
@@ -21,9 +21,11 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\Extensibility\nunit.extensibility\nunit.extensibility.csproj" />
+    <ProjectReference Condition="'$(TargetFramework)'=='net8.0'"
+                      Include="..\..\NUnitCommon\nunit.agent.core\nunit.agent.core.csproj" />
```

Comment:
Engine references agent? But only for .net8.0.

Reply:
Yes... because the .NET 8.0 engine only runs in process, making it a sort of agent in itself. I want to eventually create an in-process agent, at which point we can drop the reference.